<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BGLR-RAJESHV" name="Default test" tests="8" failures="1" timestamp="28 Mar 2014 19:15:33 GMT" time="183.858" errors="0">
  <testcase name="chkInactive" time="26.749" classname="ethos.test.VolumeShapeOutput"/>
  <testcase name="chkResetBtn" time="13.855" classname="ethos.test.VolumeShapeOutput">
    <failure type="java.lang.AssertionError" message="Reset button didn&amp;apos;t reset the values">
      <![CDATA[java.lang.AssertionError: Reset button didn't reset the values
	at com.thoughtworks.selenium.SeleneseTestBase.fail(SeleneseTestBase.java:391)
	at com.thoughtworks.selenium.SeleneseTestBase.assertTrue(SeleneseTestBase.java:395)
	at ethos.test.VolumeShapeOutput.chkResetBtn(VolumeShapeOutput.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:702)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:87)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- chkResetBtn -->
  <testcase name="findDeliveryPoint" time="15.44" classname="ethos.test.VolumeShapeOutput"/>
  <testcase name="intervalDataShapeOutputGrid" time="10.215" classname="ethos.test.VolumeShapeOutput"/>
  <testcase name="intervalDataShapeOutputMeter" time="8.827" classname="ethos.test.VolumeShapeOutput"/>
  <testcase name="tickAllUntickAll" time="19.564" classname="ethos.test.VolumeShapeOutput"/>
  <testcase name="verifyProductGrp" time="0.515" classname="ethos.test.VolumeShapeOutput"/>
  <testcase name="verifyProductInProdGrps" time="10.738" classname="ethos.test.VolumeShapeOutput"/>
</testsuite> <!-- Default test -->
